app/api/helpers/mail.py

```diff
@@ -197,12 +197,17 @@ def send_email_session_state_change(email, session, mail_override: Dict[str, str
         logger.error('No mail found for session state change: ' + session.state)
         return
 
+    if mail.get('bcc'):
+        bcc = list(set(organizers_email + mail.get('bcc')))
+    else:
+        bcc = organizers_email
```

Comment:
```suggestion
    bcc = list(set(organizers_email + mail.get('bcc', [])))
```

Reply:
sir this is giving error.
error when mail.bcc is empty---> cannot add list with nonetype.